pages/api/exportStats.ts

```diff
@@ -0,0 +1,56 @@
+import { withApiAuth } from "../../lib/auth/withApiAuth";
+import { withApiMethods } from "../../lib/withApiMethods";
+import { prisma } from "../../lib/db";
+const excel = require("excel4node");
+
+export default withApiAuth(
+  withApiMethods({
+    GET: async (_req, res) => {
+      try {
+        const allLetters = await prisma.letter.findMany();
+        const allAuthors = await prisma.author.findMany();
+        const allDestinations = await prisma.destination.findMany();
+
+        console.log(allAuthors, allDestinations);
+        const workbook = new excel.Workbook();
+        const worksheet1 = workbook.addWorksheet("Letters");
+
+        const headingStyle = workbook.createStyle({
+          font: {
+            bold: true,
+            size: 14,
+          },
+        });
+
+        const normalStyle = workbook.createStyle({
+          font: {
+            bold: false,
+            size: 12,
+          },
+        });
+
+        worksheet1.cell(1, 1).string("Class name").style(headingStyle);
+        worksheet1.cell(1, 2).string("Register number").style(headingStyle);
+        worksheet1.cell(1, 3).string("Destination").style(headingStyle);
```

Comment:
I'd separate the whole stats exporting logic to a separate file tho. And to some separate functions.

The `/lib/statsExport` would be a perfect home for this code